src/snowflake/cli/api/project/schemas/entities/common.py

```diff
@@ -162,3 +162,76 @@ def get_secrets_sql(self) -> str | None:
             return None
         secrets = ", ".join(f"'{key}'={value}" for key, value in self.secrets.items())
         return f"secrets=({secrets})"
+
+
+class ProcessorMapping(UpdatableModel):
```

Comment:
From here until EOF is the real change, everything else just in support for this move.